### PR TITLE
[bump] orchestra v0.0.4

### PR DESCRIPTION
**Pull request by drahnr**
Compatibility for the next orchestra release, assuming https://github.com/paritytech/orchestra/pull/15 will be merged

CC @sandreim @vstakhov

### CODE REVIEW

**Comment by sandreim** [comment]
The next release needs to also bring this fix: https://github.com/paritytech/orchestra/pull/17

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2163238

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2163333

File: node/service/src/db/mod.rs [hunk 1]
```diff
@@ -0,0 +1,158 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```
[the remark this answers is not in the document]
Comment (drahnr) [reply node/service/src/db/mod.rs:1]: Jesus, that's from the processbot. Why on earth is that using the merge strategy and not rebase?

[the remark this answers is not in the document]
Comment (vstakhov) [reply node/service/src/db/mod.rs:1]: Or from somehow too wide `git add` :)

[the remark this answers is not in the document]
Comment (drahnr) [reply node/service/src/db/mod.rs]: or that

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2163458

**Review by sandreim — APPROVED** [review]

**Comment by vstakhov** [comment]
It seems that some of the overseer tests are broken.

**Comment by drahnr** [comment]
> It seems that some of the overseer tests are broken.

The only behavior change is https://github.com/paritytech/orchestra/pull/17 but I won't have any cap left to do the investigation in 2022

**Review by vstakhov — APPROVED** [review]

**Comment by vstakhov** [comment]
bot merge

**Pull request merged**